Workflow failure: [PR #1171] - run-acceptance-tests

## Workflow Failure

Triggered by PR: https://github.com/pulumi/pulumi-gitlab/pull/1171

[run-acceptance-tests](https://github.com/pulumi/pulumi-gitlab/blob/master/.github/workflows/run-acceptance-tests.yml) has failed. See the list of failures below:

- [2026-04-18T04:14:26.000Z](https://github.com/pulumi/pulumi-gitlab/actions/runs/24596466650)
  - ❌ [lint / lint](https://github.com/pulumi/pulumi-gitlab/actions/runs/24596466650/job/71927799582): lint (failure)

- [2026-04-19T04:17:01.000Z](https://github.com/pulumi/pulumi-gitlab/actions/runs/24620662192)
  - ❌ [lint / lint](https://github.com/pulumi/pulumi-gitlab/actions/runs/24620662192/job/71990947121): lint (failure)

- [2026-04-20T04:21:04.000Z](https://github.com/pulumi/pulumi-gitlab/actions/runs/24648073357)
  - ❌ [lint / lint](https://github.com/pulumi/pulumi-gitlab/actions/runs/24648073357/job/72065313442): lint (failure)

- [2026-04-21T04:17:00.000Z](https://github.com/pulumi/pulumi-gitlab/actions/runs/24703538561)
  - ❌ [lint / lint](https://github.com/pulumi/pulumi-gitlab/actions/runs/24703538561/job/72252264023): lint (failure)

Error ([link](https://github.com/pulumi/pulumi-gitlab/actions/runs/24648073357/job/72065313442#step:7:36)):

```
Error: can't load config: the Go language version (go1.25) used to build golangci-lint is lower than the targeted Go version (1.26.1)
The command is terminated due to an error: can't load config: the Go language version (go1.25) used to build golangci-lint is lower than the targeted Go version (1.26.1)
make: *** [Makefile:228: lint] Error 3
```

Same golangci-lint Go version mismatch as pulumi-ec. Fix: [https://github.com/pulumi/ci-mgmt/pull/2130](https://github.com/pulumi/ci-mgmt/pull/2130)

--- a/mise.toml
+++ b/mise.toml
@@ -1,5 +1,4 @@
 [tools]
-golangci-lint = "2.7.0"
 "vfox-pulumi:pulumi/pulumi-std" = "2.2.0"
 "vfox-pulumi:pulumi/pulumi-converter-terraform" = "1.2.4"
 "vfox-pulumi:pulumi/pulumi-local" = "0.1.6"

